src/shipit_static_analysis/shipit_static_analysis/cli.py

```diff
@@ -111,6 +111,9 @@ def main(phabricator,
                 error=e,
             )
 
+            # Then raise to mark task as erroneous
+            raise
```

Comment:
Isn't a raised exception going to be logged on papertrail?

Reply:
Badly formatted, and i got a lot of missing data for static analysis on papertrail.
This is more to trigger sentry & taskcluster errors